target/linux/ramips/image/mt7621.mk

```diff
@@ -915,6 +982,25 @@ define Device/d-team_pbr-m1
 endef
 TARGET_DEVICES += d-team_pbr-m1
 
+define Device/dna_valokuitu-plus-ex400
+  $(Device/dsa-migration)
+  IMAGE_SIZE := 117m
+  PAGESIZE := 2048
+  MKUBIFS_OPTS := --min-io-size=$$(PAGESIZE) --leb-size=124KiB --max-leb-cnt=96 \
+  		  --log-lebs=2 --space-fixup --squash-uids
+  DEVICE_VENDOR := DNA
+  DEVICE_MODEL := Valokuitu Plus EX400
+  IMAGES := factory.ubifs sysupgrade.tar
+  IMAGE/factory.ubifs := kernel-initramfs-bin | append-dtb | lzma | uImage lzma | \
+                         dna-bootfs with-initrd | dna-header | \
+                         append-md5sum-ascii-salted
+  IMAGE/sysupgrade.tar := dna-bootfs | sysupgrade-tar kernel=$$$$@ | check-size | \
+  			  append-metadata
+  DEVICE_IMG_NAME = $$(DEVICE_IMG_PREFIX)-$$(2)
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615-firmware kmod-usb3
```

Comment:
Device has usb2 only as far as I know, so maybe kmod-usb2 is enough (I have  not checked, just wondering)